mittab/apps/tab/fixtures/testing_db.json

```diff
@@ -1 +1 @@
me": "GW Roundup", "debaters": [95, 97], "checked_in": true}}, {"pk": 51, "model": "tab.team", "fields": {"school": 8, "seed": 0, "name": "NYU C", "debaters": [101, 103], "checked_in": false}}, {"pk": 52, "model": "tab.team", "fields": {"school": 27, "seed": 1, "name": "AU Patrick Bateman", "debaters": [102, 104], "checked_in": true}}, {"pk": 54, "model": "tab.team", "fields": {"school": 8, "seed": 0, "name": "NYU D", "debaters": [107, 108], "checked_in": false}}, {"pk": 55, "model": "tab.team", "fields": {"school": 13, "seed": 1, "name": "GW B", "debaters": [109, 110], "checked_in": true}}, {"pk": 56, "model": "tab.team", "fields": {"school": 27, "seed": 0, "name": "AU Elle Woods", "debaters": [112, 114], "checked_in": true}}, {"pk": 57, "model": "tab.team", "fields": {"school": 11, "seed": 0, "name": "Stanford Rall Bad Bitchezzz", "debaters": [111, 113], "checked_in": true}}, {"pk": 58, "model": "tab.team", "fields": {"school": 27, "seed": 0, "name": "AU Frasier Crane", "debaters": [117, 119], "checked_in": true}}, {"pk": 59, "model": "tab.team", "fields": {"school": 13, "seed": 3, "name": "GW Last SOTY/Penultimate SOTY", "debaters": [115, 116], "checked_in": true}}, {"pk": 60, "model": "tab.team", "fields": {"school": 12, "seed": 0, "name": "Penn JT", "debaters": [118, 126], "checked_in": true}}, {"pk": 61, "model": "tab.team", "fields": {"school": 27, "seed": 0, "name": "AU Harvey Specter", "debaters": [122, 124], "checked_in": true}}, {"pk": 62, "model": "tab.team", "fields": {"school": 15, "seed": 0, "name": "Smith Gryffindor", "debaters": [121, 123], "checked_in": true}}, {"pk": 63, "model": "tab.team", "fields": {"school": 12, "seed": 1, "name": "Penn MW", "debaters": [125, 333], "checked_in": true}}, {"pk": 65, "model": "tab.team", "fields": {"school": 12, "seed": 0, "name": "Penn DS", "debaters": [131, 132], "checked_in": true}}, {"pk": 66, "model": "tab.team", "fields": {"school": 15, "seed": 0, "name": "Smith Ravenclaw", "debaters": [128, 129], "checked_in": true}}, {"pk": 67, "model": "tab.team", "fields": {"school": 15, "seed": 1, "name": "Smith Slytherin", "debaters": [133, 134], "checked_in": true}}, {"pk": 68, "model": "tab.team", "fields": {"school": 14, "seed": 0, "name": "Tufts EK", "debaters": [135, 136], "checked_in": true}}, {"pk": 69, "model": "tab.team", "fields": {"school": 23, "seed": 1, "name": "Georgetown Bend and Snap", "debaters": [137, 138], "checked_in": true}}, {"pk": 70, "model": "tab.team", "fields": {"school": 14, "seed": 1, "name": "Tufts ES", "debaters": [139, 142], "checked_in": true}}, {"pk": 71, "model": "tab.team", "fields": {"school": 16, "seed": 1, "name": "W&M Django's Novices", "debaters": [140, 141], "checked_in": true}}, {"pk": 72, "model": "tab.team", "fields": {"school": 14, "seed": 0, "name": "Tufts Bendrew Husick", "debaters": [143, 146], "checked_in": false}}, {"pk": 73, "model": "tab.team", "fields": {"school": 42, "seed": 1, "name": "W&M/UAlbany Hybrid", "debaters": [144, 145], "checked_in":
```

Comment:
Any idea why this file was cleared? It contains all of the teams used to test the pairing logic, and it seems like it was wiped to a blank slate